### PR TITLE
Extend documentation for all control files

### PR DESCRIPTION
**Pull request by supertom01**
This pull request adds more extensive documentation and some todo's to the control logic code of this microcontroller.

### CODE REVIEW

File: Core/Inc/Control/stateEstimation.h [hunk 1]
```diff
@@ -22,27 +22,48 @@
...
-	float dribbleSpeedBeforeGotBall;
+	float visionYaw;					// The yaw for this robot as indicated by vision
+	bool visionAvailable;				// Wether vision data can be used at this point
+	float xsensAcc[2];					// The acceleration as measured by the IMU in the X and Y directions
+	float xsensYaw;						// They yaw for this robot as indicated by the IMU
+	float rateOfTurn;					// [rad/s]
+	float wheelSpeeds[4];				// The speed for each wheel 
+	float dribblerSpeed;				// The measured speed of the dribbler
+	float dribblerFilteredSpeed;		// The filtered speed of the dribbler
+	float dribbleSpeedBeforeGotBall;	// The speed of the dribbler before it had a ball
```
Comment (emielsteerneman) [inline Core/Inc/Control/stateEstimation.h:33]: We should really convert this entire struct into a REM message

File: Core/Src/Control/control_util.c [hunk 2]
```diff
@@ -2,7 +2,7 @@
 #include "control_util.h"
 
 void control_util_Init() {
-	bool MOTORS_50W = true; // Keep this on the offchance that we're going to use the 30W motors again
+	bool MOTORS_50W = true; // Keep this on the in case that we're going to use the 30W motors again
```
Comment (emielsteerneman) [inline Core/Src/Control/control_util.c:5]: We should ditch this variable

**Review by emielsteerneman — APPROVED** [review]
Lgtm

**Comment by supertom01** [comment]
Yup the code will indeed need a cleanup

**Pull request merged**